src/domain/transaction/mod.rs

```diff
@@ -444,7 +444,13 @@ impl RelayerTransactionFactory {
             NetworkType::Stellar => {
                 let signer_service =
                     Arc::new(StellarSignerFactory::create_stellar_signer(&signer)?);
-                let stellar_provider = StellarProvider::new(&relayer.network)
+
+                let network = match StellarNetwork::from_network_str(&relayer.network) {
```

Comment:
Same as above, we should apply logic that is part od active PR. Let's create follow up ticket for that.

Reply:
https://linear.app/openzeppelin-development/issue/PLAT-6624/implement-new-rpcconfig